OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -42,6 +42,8 @@ public sealed class WeaponInfo
 		[Desc("The sound played when the weapon is fired.")]
 		public readonly string[] Report = null;
 
+		public readonly bool OnlyPlayReportOnFirstBurst = false;
```

Comment:
IMO you are both right.  This is a bit of a nasty hack, but so is the suggested alternative.

I propose an option c:
```csharp
[Desc("The sound played when the first shot in a burst is fired.")]
public readonly string[] BurstReport = null;
```